src/Microsoft.Health.Core/Microsoft.Health.Core.csproj

```diff
@@ -1,34 +1,36 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
-    <PropertyGroup>
-        <TargetFrameworks>$(SupportedFrameworks);</TargetFrameworks>
-    </PropertyGroup>
+  <PropertyGroup>
+    <Description>Common primitives and utilities used by Microsoft Health.</Description>
+    <TargetFrameworks>$(SupportedFrameworks);</TargetFrameworks>
+  </PropertyGroup>
 
-    <ItemGroup>
-        <PackageReference Include="Microsoft.Extensions.Hosting.Abstractions" Version="$(SdkPackageVersion)" />
-        <PackageReference Include="Microsoft.Extensions.Logging" Version="$(SdkPackageVersion)" />
-        <PackageReference Include="Ensure.That" Version="9.2.0" />
-        <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
-        <PackageReference Include="Newtonsoft.Json.Schema" Version="3.0.13" />
-    </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Extensions.Hosting.Abstractions" Version="$(SdkPackageVersion)" />
```

Comment:
Re: `Normalize .csproj identation to 2 spaces`
What is the default? Will this play nice with everyone's editors, is there a settings file to make this consistent?

Reply:
It should, as this is the default (and most of the other files were using this). However, we may want to consider the use of an *.editorconfig* file in the near future, perhaps as part of another PR.